src/libs/routes/authMiddleWare.ts

```diff
@@ -16,15 +16,15 @@ const authMiddleware = (module, permissionType) => async (req, res, next) => {
     user = jwt.verify(token, secret);
   } catch (err) {
     next({
-      err: 'Unauthorized',
+      err: 'Unauthorized (1)',
```

Comment:
key must error and do not extra characters like (1).
Use proper status code or message
see http status code